iceoryx_posh/include/iceoryx_posh/popo/subscriber.hpp

```diff
@@ -143,11 +143,17 @@ class Subscriber_t
     /// @brief Unset the semaphore if one is set
     void unsetChunkReceiveSemaphore() noexcept;
 
+    /// @brief Get the service description of this subscriber.
+    capro::ServiceDescription getServiceDescription() const;
+
```

Comment:
Great stuff! :) I missed this sometimes.

Reply:
Fixed.